x-pack/platform/plugins/private/observability_ai_assistant_management/public/routes/components/settings_page.tsx

```diff
@@ -24,26 +35,84 @@ export function SettingsPage() {
     services: {
       application: { navigateToApp, isAppRegistered },
       serverless,
+      spaces,
+      cloud,
     },
   } = useKibana();
 
   const router = useObservabilityAIAssistantManagementRouter();
   const knowledgeBase = useKnowledgeBase();
+  const { euiTheme } = useEuiTheme();
+
+  const [currentSpaceSolution, setCurrentSpaceSolution] = useState<SolutionView>();
+
+  // Determine the current solution. For serverless projects, derive it from cloud.serverless.projectType
+  const currentSolution: SolutionView | undefined = serverless
+    ? cloud?.serverless?.projectType === 'observability'
+      ? 'oblt'
+      : 'es'
+    : currentSpaceSolution;
```

Comment:
Won't `currentSpaceSolution` be populated with the project type on serverless? Afaict `currentSpaceSolution` (of type `SolutionView` can have the following values `"classic" | "oblt" | "security" | "es" | "chat"`

Meaning we can just use `currentSpaceSolution` (and we won't need `currentSolution`)
```suggestion
  const currentSolution =  currentSpaceSolution;
```

Reply:
In Serverless, the `solution` in a space is not always defined. More details [here](https://elastic.slack.com/archives/C5TQ33ND8/p1750845883187049?thread_ts=1750776221.281699&cid=C5TQ33ND8). So we have to reply on projectType for Serverless.

I initially tested out SolutionView for both ECH and Serverless, but it only reliably works for ECH.